plugins/processors/starlark/logging.go

```diff
@@ -0,0 +1,45 @@
+package starlark
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/influxdata/telegraf/models"
+	"go.starlark.net/starlark"
+	"go.starlark.net/starlarkstruct"
+)
+
+// Module that defines all the supported logging functions
+var LogModule = &starlarkstruct.Module{
+	Name: "log",
+	Members: starlark.StringDict{
+		"debug": starlark.NewBuiltin("log.debug", log),
+		"info":  starlark.NewBuiltin("log.info", log),
+		"warn":  starlark.NewBuiltin("log.warn", log),
+		"error": starlark.NewBuiltin("log.error", log),
+	},
+}
+
+// The logger used by the module
+var logger = models.NewLogger("processors", "starlak", "logging")
```

Comment:
this is the only line that doesn't sit well with me. 
1. Ideally I'd like the alias to default to the script file name, if it's given. If not maybe we should consider suggesting setting the alias in the plugin config.
2. Loggers are typically initialized with the plugin name and allow overriding the alias. I think with some work we could inject the logger into this function.

Reply:
@ssoroka it should be better now, please check again